build-iq.sh

```diff
@@ -24,7 +24,3 @@ helm lint charts/nexus-iq
 
 # package the charts into tgz archives
 helm package charts/nexus-iq --destination docs
-
-# index the existing tgz archives
-cd docs
-helm repo index . --url https://sonatype.github.io/helm3-charts
```

Comment:
if you remove this from here, it seems that it would _not_ index when IQ releases. I think it should stay here.

Reply:
Yes that should be there -  I didn't know that was taken out.

Reply:
https://github.com/sonatype/helm3-charts/pull/245/commits/5c4fb48befcfc8fd94a7fcd8f3331064a3d76380